src/esp/scene/SceneNode.h

```diff
@@ -53,6 +54,12 @@ class SceneNode : public MagnumObject {
     return this->absoluteTransformation().translation();
   }
 
+  Magnum::Range3D meshBB_;
```

Comment:
Why are these public?

Reply:
To avoid the bloat of public getter/setter functions.

Reply:
Yeah, it does feel like it adds bloat but for consistency with the rest of the code and the [coding style](https://google.github.io/styleguide/cppguide.html#Access_Control) these should be private. Public members break encapsulation by exposing implementation details in the interface making it harder to change code and ensure interface safety. They prevent you from making accessors read-only or write-only. They also allow clients to store a pointer to such a member, making code harder to reason about.

The one case where data members should be public is for a [pure data structure](https://google.github.io/styleguide/cppguide.html#Structs_vs._Classes). In that case, the trailing under-score is omitted.

I'm hoping to add [clang-tidy](https://clang.llvm.org/extra/clang-tidy/) support soon in order to have the presubmit check for the coding style.